### DISCUSSION TITLE
Prev and next links of pages overlap when titles is too long

### DISCUSSION THREAD

**Discussion by MarshalX** [pradyunsg/furo#371] [Potential Issue] [answered]
It can be easy reproduced in "Responsive Design Mode".  Link: https://wrtc.readthedocs.io/en/latest/webrtc.interfaces.media_stream_track.html

<details>
  <summary>Screenshot from mobile Safari</summary>

  ![image](https://user-images.githubusercontent.com/15520314/152661876-edb5651e-ae20-4ab1-a444-c0351320cd0b.png)
</details>

**Comment by pradyunsg** [comment, marked as answer]
<img width="338" alt="Screenshot 2023-07-15 at 11 19 52" src="https://github.com/pradyunsg/furo/assets/3275593/4e5e74c1-6ecc-4d62-b699-d989b32aca1a">

Looks like this got resolved at some point!
